plugins/processors/aws/ec2/ec2_test.go

```diff
@@ -1,14 +1,19 @@
 package ec2
 
 import (
+	"context"
 	"testing"
 
+	"github.com/influxdata/telegraf/agenthelper"
 	"github.com/influxdata/telegraf/config"
 	"github.com/influxdata/telegraf/testutil"
 	"github.com/stretchr/testify/require"
 )
 
 func TestBasicStartup(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipped test that connects to external service")
+	}
```

Comment:
With the logic to check settings in Init() but the connection code moved to Start(), shouldn't these tests add
```
require.NoError(t, p.Init())
```
and remove the p.Start() and p.Stop() lines? Then we won't have to skip this test. Same goes for TestBasicStartupWithEC2Tags().

Reply:
Yes you are right, I've updated these.